package.json

```diff
@@ -103,6 +103,7 @@
 		"mkdirp": "0.5.1",
 		"node-sass": "4.11.0",
 		"node-watch": "0.6.0",
+		"parcel-bundler": "1.12.2",
```

Comment:
I tried using `wp-scripts` but it's not usable at the moment for this kind of use-cases as it automatically uses `wp.*` globals. In this particular case I want to bundle the dependencies.

Reply:
Should we add `--webpack-no-externals` flag to `wp-scripts build`?

Reply:
We would have to add support for CSS as well. Does Parcel cover RTL support?

Reply:
Parcel can support RTL through post-css plugins. That said, I'd prefer using `wp-scripts` but there are still things to figure out before:

 - Do we want an `--webpack-no-externals`. Is it something useful outside of this playground?
 - How do we build CSS with `wp-scripts`

Reply:
That's a great question for Core JS chat today :)

We should also check with @front team since they have this `create-clould-block` tool which scaffolds cloud blocks: 
https://github.com/front/create-cloud-block/blob/master/examples/1-simple-block/webpack.config.js

It uses different externals and bundles CSS as well.